appveyor.yml

```diff
@@ -0,0 +1,54 @@
+image: Visual Studio 2017
+
+# version format
+version: 0.0.{build}
+
+# UMBRACO_PACKAGE_PRERELEASE_SUFFIX if a rtm release build this should be blank, otherwise if empty will default to alpha
+# example UMBRACO_PACKAGE_PRERELEASE_SUFFIX=beta
+init:
+  - set UMBRACO_PACKAGE_PRERELEASE_SUFFIX=
+
+cache:
+  - src\packages -> **\packages.config  # preserve "packages" directory in the root of build folder but will reset it if packages.config is modified
+
+before_build:
+  - nuget restore src
+
+build_script:
+  - build-appveyor.cmd
+
+artifacts:
+  - path: artifacts\*.nupkg
+  - path: artifacts\*.zip
+
+deploy:
+    # MyGet Deployment for builds & releases
+  - provider: NuGet
+    server: https://www.myget.org/F/byte5digital/api/v2/package
+    symbol_server: https://www.myget.org/F/byte5digital/symbols/api/v2/package
+    api_key:
+      secure: vsHr1qAsDwPTvZV7ljoZdx2LAAuk7Xq+0vPHPFhqlPQPXinsAObb8XPLMpQoWnYW
```

Comment:
The API key will need generating from MyGet, then encrypted on AppVeyor - depending who's AppVeyor account is used to manage the CI build.  Of course this wont be known until it's all set-up/configured.